lake/ztests/vacuum-dryrun.yaml

```diff
@@ -0,0 +1,19 @@
+script: |
+  export ZED_LAKE=test
+  zed init -q
+  zed create -q -orderby x:asc test
+  zed use -q test
+  echo {x:1} | zed load -q -
+  r=$(echo {x:2} | zed load - | head -1 | awk '{print $1}')
+  r=$(zed revert $r | awk '{print $5}')
+  zed vacuum -f -dryrun
```

Comment:
Instead of adding separate tests, can't we get the same coverage by just doing `zed vacuum -f -dryrun` before `zed vacuum -f` in the existing tests?

Reply:
Yeah ok